databroker/eventsource/shim.py

```diff
@@ -24,6 +24,10 @@ class EventSourceShim(object):
     into the implementations
     '''
 
+    @property
```

Comment:
What is this for?

Reply:
it is added to be consistent with the db.ES[name] for accessing event sources

Reply:
_ArchiverReader works as a (internal/private) factory for configuring multiple archiver event sources under the Broker API. This approach has been already blessed by Tom.

Reply:
I know - but I disagree with Tom on this.

Reply:
This should probably be removed now?

Reply:
Name (in shim.py) is used by BrokerES::event_sources_by_name.